vpr/src/route/rr_graph2.h

```diff
@@ -7,6 +7,8 @@
 #include "rr_types.h"
 #include "device_grid.h"
 
+struct t_rr_graph_storage; //Forward declaration
```

Comment:
t_rr_graph_storage is a class, not a struct

Reply:
I've fixed this for consistency, although I believe there is no real distinction between struct/class (other than default members being public vs private).

Reply:
It generates warnings from the compile, which we avoid.